README.md

```diff
@@ -16,14 +16,30 @@
 
 ![preview](https://cdn.discordapp.com/attachments/343113240143986709/1328204316137226250/image.png?ex=6785da29&is=678488a9&hm=294b679aa54a290a4b05567dcd203c926d648a2c27459bba03813cd2bc4696f3&)
 
-## Notes
-- If you delete or add a `.vrm` while Desktop Mate is running, the mod will (currently) not detect this change. If you add or remove any models you will have to restart Desktop Mate.
+## Applied changes
+ - Window patch (won't show on alt+tab/win + tab)
+ - Detects models added to folder (addition/deletion of .vrm files at runtime)
+ - Saves the current model to config file to load it on boot.
 
 ## Planned Changes
 - Per-model settings
-- Detect added/removed models at runtime
 - Allow customization of buttons
 
+## Known Issues
+ - When the application launches sometimes will lose focus through the current window (it should be placed behind the current window, just minimize the current window and click over the character)
+
+## Build Guide
```

Comment:
This can be reworded and some bullet points combined into others. Also, the project directory should not be the same as or placed in the game directory.